Gemfile

```diff
@@ -91,6 +91,10 @@ gem "time", "~> 0.2.2"  # CVE-2023-28756: https://github.com/advisories/GHSA-fg7
 gem "uri",  "~> 0.13.2" # CVE-2025-27221: https://github.com/advisories/GHSA-22h5-pq3x-2gf2
                         # Avoid URI 1.0.0 for now due to: https://github.com/ruby/uri/issues/125
 
+# gems to ensure that appliances and containers have a consistent set of gems
```

Comment:
Should this be in a `production` group or something?

Reply:
so looks like we want this in development and on appliances. That may basically be "everywhere"

Reply:
Yeah, I was thinking we'd want it consistent in dev too, so "everywhere"